src/ScalarMessaging.ts

```diff
@@ -474,8 +474,8 @@ async function setBotPower(
         if (ignoreIfGreater === true) {
             // As per https://matrix.org/docs/spec/client_server/r0.6.0#m-room-power-levels
             const currentPl = (
-                powerLevels.content.users && powerLevels.content.users[userId]
-            ) || powerLevels.content.users_default || 0;
+                powerLevels.users && powerLevels.users[userId]
+            ) || powerLevels.users_default || 0;
```

Comment:
Suggestion doesn't want to work but this could be much cleaner as

```
const currentPl = powerLevels.users?.[userId] ?? powerLevels.users_default ?? 0;
```